.circleci/config.yml

```diff
@@ -51,7 +51,8 @@ commands:
             ddev --version
             ddev config global --instrumentation-opt-in=false
             ddev config --project-name test-$CIRCLE_WORKFLOW_JOB_ID --project-type drupal9 --docroot docroot
-            ddev add-on get https://github.com/GetDKAN/ddev-dkan/archive/refs/heads/<< parameters.addon_branch >>.tar.gz
+            wget -P ../ https://github.com/GetDKAN/ddev-dkan/archive/refs/heads/<< parameters.addon_branch >>.tar.gz
+            ddev add-on get ../<< parameters.addon_branch >>.tar.gz
```

Comment:
For some reason, add-on get stopped working with a github url, even though curl and wget can download the file just fine. Had to switch to this.